test-app/tests/integration/components/bs-tab-test.gjs

```diff
@@ -274,73 +329,89 @@ module('Integration | Component | bs-tab', function (hooks) {
   });
 
   test('calls onChange when changing active tab', async function (assert) {
-    let action = sinon.spy();
-    this.actions.change = action;
-
-    await render(hbs`<BsTab @fade={{false}} @onChange={{action 'change'}} as |tab|>
-  <tab.pane @id='pane1' @title='Tab 1'>
-    tabcontent 1
-  </tab.pane>
-  <tab.pane @id='pane2' @title='Tab 2'>
-    tabcontent 2
-  </tab.pane>
-</BsTab>`);
+    const onChange = sinon.spy();
+
+    await render(
+      <template>
+        <BsTab @fade={{false}} @onChange={{onChange}} as |tab|>
+          <tab.pane @id='pane1' @title='Tab 1'>
+            tabcontent 1
+          </tab.pane>
+          <tab.pane @id='pane2' @title='Tab 2'>
+            tabcontent 2
+          </tab.pane>
+        </BsTab>
+      </template>,
+    );
 
     await click('ul.nav.nav-tabs li:nth-child(2) a');
-    assert.ok(action.calledWith('pane2', 'pane1'));
+    assert.ok(onChange.calledWith('pane2', 'pane1'));
 
     assertActiveTab.call(this, assert, 0, false);
     assertActiveTab.call(this, assert, 1, true);
   });
 
   test('when onChange returns false active tab is not changed', async function (assert) {
-    let action = sinon.stub();
-    action.returns(false);
-    this.actions.change = action;
-
-    await render(hbs`<BsTab @fade={{false}} @onChange={{action 'change'}} as |tab|>
-  <tab.pane @id='pane1' @title='Tab 1'>
-    tabcontent 1
-  </tab.pane>
-  <tab.pane @id='pane2' @title='Tab 2'>
-    tabcontent 2
-  </tab.pane>
-</BsTab>`);
+    const onChange = sinon.stub().returns(false);
+
+    await render(
+      <template>
+        <BsTab @fade={{false}} @onChange={{onChange}} as |tab|>
+          <tab.pane @id='pane1' @title='Tab 1'>
+            tabcontent 1
+          </tab.pane>
+          <tab.pane @id='pane2' @title='Tab 2'>
+            tabcontent 2
+          </tab.pane>
+        </BsTab>
+      </template>,
+    );
 
     await click('ul.nav.nav-tabs li:nth-child(2) a');
-    assert.ok(action.calledWith('pane2', 'pane1'));
+    assert.ok(onChange.calledWith('pane2', 'pane1'));
 
     assertActiveTab.call(this, assert, 0, true);
     assertActiveTab.call(this, assert, 1, false);
   });
 
   test('changing active tab does not change public activeId property (DDAU)', async function (assert) {
-    this.set('paneId', 'pane1');
-    await render(hbs`<BsTab @fade={{false}} @activeId={{this.paneId}} as |tab|>
-  <tab.pane @id='pane1' @title='Tab 1'>
-    tabcontent 1
-  </tab.pane>
-  <tab.pane @id='pane2' @title='Tab 2'>
-    tabcontent 2
-  </tab.pane>
-</BsTab>`);
+    class State {
+      @tracked paneId = 'pane1';
```

Comment:
I _think_ this doesn't need to be tracked. We assert that it isn't changed. If it would be changed, the test would fail. We don't care if UI would be in sync with the state. Or do I miss something?